src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -105,7 +105,7 @@ public void archivePerson(Person toArchive, ArchivedPersonList archivedPersonLis
         if (!internalList.remove(toArchive)) {
             throw new PersonNotFoundException();
         }
-        archivedPersonList.addArchivedPerson(toArchive);
+        // archivedPersonList.addArchivedPerson(toArchive);              // Uncomment for future iteration
```

Comment:
Perhaps format this as a TODO?

Reply:
Hi Isaac, I've uncommented this line so that it works as expected and it passes test cases based on the archive functionality.